stdlib/test/builtin/test_simd.mojo

```diff
@@ -325,6 +325,43 @@ def test_shift():
     )
 
 
+def test_shuffle():
+    print("== test_shuffle")
```

Comment:
**Suggestion** You can drop the `print` as it's only needed for `FileCheck`-based tests. Ditto for `test_join`, etc.

Reply:
I removed the prints.  The new location information for asserts is great!